pkgs/applications/video/handbrake/default.nix

```diff
@@ -109,9 +104,12 @@ stdenv.mkDerivation rec {
     (if stdenv.isDarwin then "--disable-xcode"             else "")
   ];
 
-  # NOTE: 2018-12-27: Check NixOS HandBrake test if changing
+  # NOTE 1: 2018-12-27: Check NixOS HandBrake test if changing
+  # NOTE 2: We have to patch libdl in to the ghb linking step because it is
```

Comment:
Could you move the comment just before the `-ldl`? Also mention that it will be fixed in next release.

Reply:
I removed my comment. I'm not familiar with plans for fixing before the next release.

Reply:
It was fixed in https://github.com/HandBrake/HandBrake/commit/f7bc55eb2cf06759b2cbe339c858d1d6a6b2d831#diff-3bfc7b7ac6e2f746f2fa2173192d96fdR253.

Reply:
Ah, thank you!